docs/source/etcd-data-model.rst

```diff
@@ -110,6 +118,13 @@ The various properties in this object have the following meanings:
   this endpoint. Each profile is applied to packets in the order that they
   appear in this list.
 
+``ipv4_nat``
+  a list of 1:1 NAT mappings to apply to the endpoint.  Inbound connections to
+  out_ip will be forwarded to in_ip.  Connections initiated from in_ip will
+  not have their source address changed, except when an endpoint attempts to
+  connect one if it's own out_ips.  Each in_ip must be associated with the
```

Comment:
Right. I think I understand the idea, it's just the "one if it's own out_ips" that's confusing me. It seems to be that it should be

> one of its own out_ips

Is that right, or am I still misunderstanding?